inputfiles/overridingTypes.jsonc

```
@@ -2792,6 +2792,16 @@
                         }
                     }
                 }
+            },
+            "HTMLMediaElement": {
+                "properties": {
+                    "property": {
+                        "preload": {
+                            "name": "preload",
+                            "overrideType": "\"none\" | \"metadata\" | \"auto\""
```

```suggestion
                            "overrideType": "\"none\" | \"metadata\" | \"auto\" | \"\""
```

Since empty string is also valid per both the MDN document and the spec: https://html.spec.whatwg.org/multipage/media.html#attr-media-preload

---

Neither [Apple](https://developer.apple.com/documentation/webkitjs/htmlmediaelement/1633059-preload) nor [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement#properties) lists it, so I left it out. Next time I'll check the spec. :-)